### PR TITLE
[docs] [wni] [aws] Update wni create command with updated flags

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
This commit adds the private-key flag to the wni aws create
command

### CODE REVIEW

File: tools/windows-node-installer/README.md [hunk 1]
```diff
@@ -91,4 +91,4 @@ Sample Delete Command:
 ```bash
 ./wni azure destroy --kubeconfig ~/OpenShift/azure/auth/kubeconfig --credentials ~/.azure/osServicePrincipal.json \
 --dir ./windowsnodeinstaller/
-```
\ No newline at end of file
+```
```
Comment (aravindhp) [inline tools/windows-node-installer/README.md]: What is going on here?

Comment (ravisantoshgudimetla) [reply tools/windows-node-installer/README.md]: Problem with the ``` which I fixed.

File: tools/windows-node-installer/README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Available Commands:
 ```bash
 ./wni aws create --kubeconfig <path to OpenShift cluster>/kubeconfig --credentials <path to aws>/credentials 
 --credential-account default --image-id ami-06a4e829b8bbad61e --instance-type m4.large --ssh-key <name of the 
-existing ssh key, ie: libra>
+existing ssh key, ie: libra> --private-key <private key to decrypt the aws instance password>
```
Comment (sebsoto) [inline tools/windows-node-installer/README.md]: If libra is being EOL then shouldnt it be changed here?

Comment (aravindhp) [reply tools/windows-node-installer/README.md]: That is good point, @sebsoto. @ravisantoshgudimetla I think we should just remove `libra`. This is internal stuff.

Comment (ravisantoshgudimetla) [reply tools/windows-node-installer/README.md]: I think it should be a separate PR or if you want badly I can do it in a separate commit within this PR.

Comment (aravindhp) [reply tools/windows-node-installer/README.md]: I agree. You should then also remove the change related to ```.

**Comment by sebsoto** [comment]
whoops, put a hold on this since I though @aravindhp wasnt done reviewing.
/hold cancel

**Pull request merged**